compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/GraphicsLayerOwnerLayer.skiko.kt

```diff
@@ -175,10 +175,14 @@ internal class GraphicsLayerOwnerLayer(
 
         mutatedFields = scope.mutatedFields
         if (maybeChangedFields != 0 || outlineChanged) {
-            invalidate()
+            triggerRepaint()
         }
     }
 
+    private fun triggerRepaint() {
```

Comment:
Having 2 functions `triggerRepaint` and `invalidate` complicates understanding of their purpose and what exactly happens.

As far as I know, `OwnedLayer.invalidate` should trigger only repaint, not layout or recomposition. Because of this, not sure why we need a separate `triggerRepaint`, which does almost the same, except it doesn't mark `isDirty = true`?

What difference does it make? Why we don't need to set it in `move` and `updateLayerProperties`? Why we need to set it in `resize` and from external calls?

Reply:
After understanding it more:
- `invalidate` asks "redraw its content" and to "schedule a new redraw"
- `triggerRepaint` asks to "schedule a new redraw"
Am I correct?

Reply:
If so, this can be a naming issue. And I suggest:
- don't create `triggerRepaint`
- rename `layerManager.invalidate` to `layerManager.requestDraw`

Reply:
> Am I correct?

Yes.

> Having 2 functions triggerRepaint and invalidate complicates understanding of their purpose and what is exactly happenning.

I'd stick with this pattern from android variant - it makes things easier to adopt the changes

Reply:
Ok, but let's add a comment what these functions do